### ISSUE TITLE
[Accessibility][Supporting the platform]: Link controls adopts disable text color in  high contrast black/white mode.

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10597]
Tracking all of the issues brought up with the 1/25 accessibility pass on Nav control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B8b59713f-87bb-4a5e-ad92-2b7b3a0ef589%7D&action=default&uid=%7B8B59713F-87BB-4A5E-AD92-2B7B3A0EF589%7D&ListItemId=1721&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 

This issue is tracking a single entry from #7803

**Comment by khmakoto** [comment]
This issue does not reproduce anymore as of version 7.50.0 as can be seen below:

![image](https://user-images.githubusercontent.com/7798177/66875171-36fe9c00-ef62-11e9-9582-afd5fc8ae1bd.png)

The links with disabled text color are actually disabled. Given that it doesn't reproduce anymore I'll close out this bug.
